docs/02-tutorial.mdx

```diff
@@ -0,0 +1,281 @@
+---
+sidebar_position: 2
+title: Tutorial
+tags: [quickstart]
+---
+
+:::info
+
+This tutorial doesn’t assume any existing knowledge of the blindnet devkit or even any privacy-related technologies.
+
+:::
+
+The purpose of the following tutorial is to serve as an entry point for the blindnet devkit, by simply walking you through its different components to give you a better understanding of the overall structure and logic.
+
+Doing so, we'll answer the following questions:
+
+- What is the purpose of the Privacy Computation Engine;
+- How to add Privacy Requests features to your Web page or applications in a snap;
+- How to easily add privacy requests management features to your "back-office" web page or applications;
+- How to automatically update stored data when managing Privacy Requests.
+
+:::tip Further Readings
+
+We strongly recommend reading the linked sections of the documentation for more details.
+
+:::
+
+## 1. Initial Configuration
+
+The [Privacy Computation Engine](./computation) is the core of blindnet devkit.
+It is a simple service you can interact with via a Rest API to create and track your users' [Privacy Requests](/docs/references/lexicon#privacy-request).
+
+Using the Privacy Computation Engine goes into three steps:
+
+1. **[App Management](/docs/access-management/set-up)**: register and obtain an application id;
+2. **[Configuration](./computation/configuration)**: configure the service using its configuration API;
+3. **Development**: call the Privacy Computation Engine from your registered application to handle Privacy Requests
+
+Step _(1) App Management_ and _(2) Configuration_ don't need to be addressed for this tutorial.
+
+:::tip Pre-configured Instance
+
+To make things easier for this tutorial, we have already pre-configured a dedicated custom instance for you to play with:
+
+<div class="text--center">
+
+**`https://devkit-pce-staging.azurewebsites.net/v0/`**
+
+</div>
+
+:::
+
+Step _(3) Development_ can be achieved directly by calling the standalone Privacy Computation Engine [REST API](https://devkit-pce-staging.azurewebsites.net/swagger/).
+
+However, the true strength of the blindnet devkit comes when combining its various components, thus drastically improving your developer experience and productivity while giving you access to additional features.
+
+Starting with our premade user interfaces.
+
+<!-- ## 2. Capture Data -->
+<!-- TODO: add information on how to capture data without the Capture Componet, i.e. call the demo storage service API to save a data from a simple HTML form -->
+
+## 2. Add Privacy-Rights Management Capabilities to Your Application
+
+User interfaces for the blindnet devkit (aka _[Privacy Components for the Web](https://github.com/blindnet-io/privacy-components-web)_) are delivered as standard Web Components available on both npm and a CDN.
```

Comment:
Would it be possible to provide a link for Web Components, to a page (maybe wikipedia) where it is explained what they are?

Reply:
Addressed in https://github.com/blindnet-io/blindnet.dev/pull/40